fem/gridfunc.cpp

```diff
@@ -1731,6 +1729,13 @@ void GridFunction::GetElementAverages(GridFunction &avgs) const
    }
 }
 
+void GridFunction::GetElementDofs(int el, Vector &dof_vals) const
+{
+   Array<int> dof_idx;
+   fes->GetElementVDofs(el, dof_idx);
```

Comment:
This is different than the original (which calls `GetElementDofs`), is that intentional?

Reply:
It seems to be that vdofs is almost always the more useful result. I'm not sure I see the use-case here for `GetElementDofs` for a non-scalar grid function (for scalar they should do the same thing).